package.json

```diff
@@ -62,5 +62,10 @@
     "hooks": {
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
     }
+  },
```

Comment:
Why are these dependencies added here?

Reply:
the dependencies were added by accident (removed in last commit)